app/services/edge_cache/bust_comment.rb

```diff
@@ -0,0 +1,32 @@
+module EdgeCache
+  class BustComment < Bust
+    def self.call(commentable)
+      return unless commentable
+
+      bust_article_comment(commentable) if commentable.is_a?(Article)
+      commentable.touch(:last_comment_at) if commentable.respond_to?(:last_comment_at)
+
+      bust("#{commentable.path}/comments/")
+      bust(commentable.path.to_s)
+
+      commentable.comments.includes(:user).find_each do |comment|
+        bust(comment.path)
+        bust("#{comment.path}?i=i")
+      end
+
+      bust("#{commentable.path}/comments/*")
+    end
+
+    # bust commentable if it's an article
+    def self.bust_article_comment(commentable)
+      bust("/") if Article.published.order(hotness_score: :desc).limit(3).ids.include?(commentable.id)
+
+      if commentable.decorate.cached_tag_list_array.include?("discuss") &&
```

Comment:
How about moving this check into the decorator? Something like 

```ruby
commentable.decorate.discussion?
```

or even more general

```ruby
commentable.decorate.short_lived?
```

Reply:
Can we hold off on this? I'm trying to make as few changes as possible in each PR so that it's easier to debug if something goes wrong. If I make this change now, I'll have to debug this refactor if something goes wrong vs comparing a copy/paste.

Also, this suggestion means I'd have to update _each_ decorator that qualifies as "commentable" - Article, PodcastEpisode, etc. I'd rather make that a more contained change in its own PR so it stretches the scope of this PR wider.